.circleci/config.yml

```diff
@@ -183,17 +183,17 @@ workflows:
           matrix:
             parameters:
               py_version: ["3.6", "3.7", "3.8"]
-              nox_session: ["lint test_core", "lint_plugins test_plugins", "test_jupyter_notebooks"]
+              nox_session: ["lint", "lint_plugins", "test_core", "test_plugins", "test_jupyter_notebooks"]
```

Comment:
so actually these are all quick, I think the heavy one is test_plugins. this should be the prize.
It's probably not worth splitting the rest into so many small chunks.